nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/Dataset.java

```diff
@@ -321,6 +309,22 @@ public Dataset(String title, int[] dimSizes) throws DatasetException {
         } catch (IOException ioe) {
             throw new DatasetException("Can't create dataset " + ioe.getMessage());
         }
+        if (addFile) {
+            addFile(title);
+        }
```

Comment:
The datasets add themselves to the Project...since the memory file constructor is used in processing when it is not wanted to add the dataset to the project, I added a flag so the memory file dataset can be added to the projects when needed